pkg/pillar/cmd/nim/nim.go

```diff
@@ -972,3 +990,13 @@ func isSwitch(ctx *nimContext, ifname string) bool {
 	}
 	return foundExcl
 }
+
+// send a sighup to wpa_supplicant to restart
+func restartWPAsupplicant() {
```

Comment:
I think this should be done by somehow notifying the wlan container using the /run directory, and have the script in the wlan container react to that notification and do the SIGHUP.